src/spdx_tools/spdx3/model/dataset/dataset.py

```diff
@@ -0,0 +1,56 @@
+#  SPDX-FileCopyrightText: 2023 spdx contributors
+#
+#  SPDX-License-Identifier: Apache-2.0
+from enum import Enum, auto
+from typing import Dict, Optional
+
+from spdx_tools.common.typing.dataclass_with_properties import dataclass_with_properties
+from spdx_tools.common.typing.type_checks import check_types_and_set_values
+
+
+class ConfidentialityLevelType(Enum):
+    RED = auto()
+    AMBER = auto()
+    GREEN = auto()
+    CLEAR = auto()
+
+
+class DatasetAvailabilityType(Enum):
+    DIRECT_DOWNLOAD = auto()
+    SCRAPING_SCRIPT = auto()
+    QUERY = auto()
+    CLICKTHROUGH = auto()
+    REGISTRATION = auto()
+
+
+@dataclass_with_properties
+class Dataset:
+    data_collection_process: Optional[str] = None
+    intended_use: Optional[str] = None
+    dataset_size: Optional[int] = None
+    dataset_noise: Optional[str] = None
+    data_preprocessing_steps: Optional[str] = None
+    sensors: Optional[Dict[str, str]] = None
+    known_biases: Optional[str] = None
+    sensitive_personal_information: Optional[bool] = None
+    anonymization_method_used: Optional[str] = None
+    confidentiality_level: Optional[ConfidentialityLevelType] = None
```

Comment:
These 3 properties don't have a maxCount, so I would expect lists here.

Reply:
Good point for `anonymization_method_used`; for the other two I [opened a PR](https://github.com/spdx/spdx-3-model/pull/219) as I think they actually should have a `maxCount`.